tests/sqllogictests/suites/udf_native/01_0001_udf_wasm.test

```diff
@@ -0,0 +1,63 @@
+query 
+select name, size, creator from list_stage(location => '@data/udf/wasm/test10-udf-wasm-gcd/') order by name
+----
+udf/wasm/test10-udf-wasm-gcd/test10-udf-wasm-gcd.tar.gz 8932 NULL
+udf/wasm/test10-udf-wasm-gcd/test10_udf_wasm_gcd.wasm 2937940 NULL
+
+
+statement ok
+CREATE FUNCTION wasm_gcd (INT, INT) RETURNS BIGINT LANGUAGE wasm HANDLER = 'wasm_gcd(int4,int4)->int4' AS $$@data/udf/wasm/test10-udf-wasm-gcd/test10_udf_wasm_gcd.wasm$$
```

Comment:
`AS '@data/udf/wasm/test10-udf-wasm-gcd/test10_udf_wasm_gcd.wasm'`

This is simple string of path, we don't need the CodeString

Reply:
The query is expecting a 'Codestring' input, but it is failing for a simple string.

```
Completed Http test for file: tests/sqllogictests/suites/udf_native/03_0001_udf_js.test ✅ (232.773806ms)
Running sql with http client: [select name, size, creator from list_stage(location => '@data/udf/wasm/test10-udf-wasm-gcd/') order by name] (232.416378ms)
Test finished, fail fast enabled, 1 out of 13 records failed to run
0: statement failed: Databend sqllogictests error: http query error: code: 1005, Text: error: 
  --> SQL:1:107
  |
1 | CREATE FUNCTION wasm_gcd (INT, INT) RETURNS BIGINT LANGUAGE wasm HANDLER = 'wasm_gcd(int4,int4)->int4' AS '@data/udf/wasm/test10-udf-wasm-gcd/test10_udf_wasm_gcd.wasm.zst'
  | ------                   -                                                                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ unexpected `'@data/udf/wasm/test10-udf-wasm-gcd/test10_udf_wasm_gcd.wasm.zst'`, expecting <CodeString>
  | |                        |                                                                                 
  | |                        while parsing (<arg_type>, ...) RETURNS <return_type> LANGUAGE <language> HANDLER=<handler> AS <language_codes>
  | while parsing `CREATE [OR REPLACE] FUNCTION [IF NOT EXISTS] <name> {AS (<parameter>, ...) -> <definition expr> | (<arg_type>, ...) RETURNS <return_type> LANGUAGE <language> HANDLER=<handler> ADDRESS=<udf_server_address>} [DESC = <description>]`


[SQL] CREATE FUNCTION wasm_gcd (INT, INT) RETURNS BIGINT LANGUAGE wasm HANDLER = 'wasm_gcd(int4,int4)->int4' AS '@data/udf/wasm/test10-udf-wasm-gcd/test10_udf_wasm_gcd.wasm.zst'
at tests/sqllogictests/suites/udf_native/01_0001_udf_wasm.test:7

Error: SelfError("sqllogictest failed")
```

Reply:
Ok! Let's keep this, or you can modify the parser using:

```
 ~ AS ~ ^ (#code_string | #literal_string)
```